tests/hawkey/test_reldep.cpp

```diff
@@ -42,6 +42,7 @@ START_TEST(test_reldeplist_add)
     for (int i = 0; i < count; ++i) {
         DnfReldep *reldep = dnf_reldep_list_index (obsoletes.get(), i);
         dnf_reldep_list_add (reldeplist.get(), reldep);
+        delete reldep;
```

Comment:
`DnfReldep` is incomplete type. If we want to use `delete` on it than this include is needed:
`#include "libdnf/repo/solvable/Dependency.hpp"`